src/Inventory/Asset/Unmanaged.php

```diff
@@ -341,4 +313,20 @@ public function getItemtype(): string
     {
         return \Unmanaged::class;
     }
+
+    /**
+     * @FIXME Remove method in GLPI 10.1.
+     */
+    public function getManagementPorts()
+    {
+        return [];
+    }
+
+    /**
+     * @FIXME Remove method in GLPI 10.1.
+     */
+    public function setManagementPorts(array $ports): Unmanaged
+    {
+        return $this;
```

Comment:
We should probably has a `Toolbox::deprecaated()` here.

Reply:
agree with this